releasenotes/notes/use-bound-pass-manager-in-circuitqnn-20c60effedb70d76.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fix in ``CircuitQNN`` that ensures unbound_pass_manager is called when caching
+    the QNN circuit and that ``bound_pass_manager`` is called when QNN parameters
+    are assigned.
```

Comment:
Can you please move this to features? A rephrase like:
`Added support for bound pass manager in ``CircuitQNN``. This additional transpilation step ensures ``unbound_pass_manager`` is called when caching the QNN circuit and that ``bound_pass_manager`` is called when QNN parameters are assigned.`